lisa/util/__init__.py

```diff
@@ -563,7 +564,9 @@ def find_group_in_lines(
     return result
 
 
-def deep_update_dict(src: Dict[str, Any], dest: Dict[str, Any]) -> Dict[str, Any]:
+def deep_update_dict(
+    src: Dict[str, Any], dest: Optional[Dict[str, Any]]
```

Comment:
Why make dest is none? If the dest is None, it doesn't need to update.

Reply:
reverted to original code